nifi-commons/nifi-security-utils/src/main/java/org/apache/nifi/security/util/CertificateUtils.java

```diff
@@ -625,6 +634,20 @@ public static boolean isTlsError(Throwable e) {
         }
     }
 
+    /**
+     *Extracts the common name from the given DN.
```

Comment:
Recommend adding a space after the asterisk to improve readability.
```suggestion
     * Extracts the common name from the given DN.
```